.github/workflows/manifest.yml

```diff
@@ -29,5 +29,14 @@ jobs:
           checkout-path: 'ncs/nrf'
           label-prefix: 'manifest-'
           verbosity-level: '1'
-          labels: 'manifest'
+          labels: >-
```

Comment:
Wht's up with the `>-` syntax; is that intentional?

Reply:
It denotes a multiline string.

Reply:
Did you run into errors without the chomping indicator? I'm not aware of us using it in many places; I guess the underlying tool should be able to handle a trailing newline, right?

Reply:
It works fine in my tests. `>` strips all newlines except the trailing. `>-` strips all newlines including the trailing.

https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines

Reply:
What I'm asking is if the chomping indicator (which is the YAML jargon for the `-` character in this case; see the YAML spec for details) is required. If not, I think it's weird to include it here and therefore bad for readability.

Reply:
Sorry, I misunderstood the jargon.
You're right, a newline should be fine. strip() defaults to removing whitespace according to the docs. I added the `-` since a newline is not called for here, without knowing any convention. I'll remove it.

    labels = [x.strip() for x in args.labels.split(',')] \
        if args.labels != 'none' else None

Reply:
Thanks!

I'm probably just hypersensitive to YAML multiline string differences because I have to periodically clean up the 'wrong' ones in the DT bindings and suggest changes in review (https://github.com/zephyrproject-rtos/zephyr/pull/36189; https://github.com/nrfconnect/sdk-nrf/pull/3576; https://github.com/zephyrproject-rtos/zephyr/pull/35357#discussion_r650289075; https://github.com/zephyrproject-rtos/zephyr/pull/35841#discussion_r648505680; ...).

There are so many ways to do it and it's nice to pick just a few if we can.